src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const queryToLowerCase = query.toLowerCase();
+
+  const visibleMovies = moviesFromServer.filter((item) => {
+    return (
```

Comment:
```suggestion
  const visibleMovies = moviesFromServer.filter((item) => (
```
you may use shorthand return here